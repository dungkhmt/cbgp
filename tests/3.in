100 100
100 208
13 4
14 5
82 10
55 68
40 38
82 74
20 50
5 65
80 20
8 73
69 5
37 9
14 29
3 6
24 62
95 12
11 65
4 1
91 8
7 24
4 2
37 72
93 73
78 71
7 8
26 85
33 60
6 2
66 32
80 85
75 17
72 69
73 91
17 94
78 6
3 65
66 69
5 100
32 5
4 15
26 80
8 77
43 48
11 64
13 92
21 4
33 80
35 32
4 27
37 2
26 20
60 26
5 66
36 65
39 31
20 22
54 70
88 89
59 14
5 2
1 15
1 33
36 40
93 8
100 72
76 55
54 81
22 1
16 25
40 49
39 30
2 3
7 4
14 17
81 94
81 75
6 56
51 58
11 34
18 17
29 59
70 81
55 32
10 4
14 18
69 2
23 47
20 1
52 98
36 38
33 58
43 13
45 21
36 5
14 81
34 54
18 28
84 4
81 17
5 25
22 32
50 22
71 2
88 58
54 75
9 72
51 10
63 69
63 99
24 67
83 18
20 32
31 25
71 1
31 16
97 3
12 13
6 96
67 86
96 56
51 88
76 68
92 21
18 19
19 47
54 57
10 74
75 14
14 34
2 96
83 14
33 26
4 58
4 12
60 80
43 21
5 61
4 89
13 95
17 29
13 21
6 97
14 90
5 38
39 5
3 96
14 54
20 35
22 71
41 36
14 11
69 87
79 30
81 29
4 33
14 36
79 25
44 52
75 64
3 78
2 9
32 61
78 2
100 69
63 24
64 54
51 89
78 36
21 27
5 9
15 2
30 36
71 32
77 73
32 78
4 69
36 11
35 44
16 5
24 8
78 97
35 46
20 33
36 76
36 42
84 45
92 12
66 61
79 31
4 24
2 65
32 46
51 4
90 54
34 64
3 36
21 12
63 46
2 1
11 5
43 92
4 88
23 19
30 5
25 30
2 72
53 51
56 2
1
32 11 90 9 153 14 34 13 118 56 154 10 61 15 65 15 82 17 28 19 98 17 49 25 47 20 115 34 34 12 124 43 114 24 113 25 111 27 5 5 57 17 86 6 110 28 51 28 121 40 18 18 58 16 108 26 117 25 129 40 125 42 173 4 22 14 130 18 3 3 154 15 82 16 116 35 126 43 118 36 121 37 120 39 50 22 0 0 48 18 178 0 109 27 50 21 119 37 29 7 26 18 2 0 92 84 128 20 164 9 152 11 122 24 23 15 116 26 19 16 119 57 60 16 101 77 101 19 97 16 119 58 47 32 162 9 75 14 121 23 88 7 80 15 70 17 42 33 103 21 156 17 70 16 161 8 126 41 18 15 120 22 43 32 107 25 46 18 17 16 45 32 74 15 25 16 26 17 123 25 72 15 50 23 69 18 114 23 46 21 152 12 155 11 4 0 94 84 79 16 
