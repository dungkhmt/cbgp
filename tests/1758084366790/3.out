(12, 3), (13, 4), (81, 9), (54, 67), (39, 37), (81, 73), (19, 49), (4, 64), (79, 19), (7, 72), (68, 4), (36, 8), (13, 28), (2, 5), (23, 61), (94, 11), (10, 64), (3, 0), (90, 7), (6, 23), (3, 1), (36, 71), (92, 72), (77, 70), (6, 7), (25, 84), (32, 59), (5, 1), (65, 31), (79, 84), (74, 16), (71, 68), (72, 90), (16, 93), (77, 5), (2, 64), (65, 68), (4, 99), (31, 4), (3, 14), (25, 79), (7, 76), (42, 47), (10, 63), (12, 91), (20, 3), (32, 79), (34, 31), (3, 26), (36, 1), (25, 19), (59, 25), (4, 65), (35, 64), (38, 30), (19, 21), (53, 69), (87, 88), (58, 13), (4, 1), (0, 14), (0, 32), (35, 39), (92, 7), (99, 71), (75, 54), (53, 80), (21, 0), (15, 24), (39, 48), (38, 29), (1, 2), (6, 3), (13, 16), (80, 93), (80, 74), (5, 55), (50, 57), (10, 33), (17, 16), (28, 58), (69, 80), (54, 31), (9, 3), (13, 17), (68, 1), (22, 46), (19, 0), (51, 97), (35, 37), (32, 57), (42, 12), (44, 20), (35, 4), (13, 80), (33, 53), (17, 27), (83, 3), (80, 16), (4, 24), (21, 31), (49, 21), (70, 1), (87, 57), (53, 74), (8, 71), (50, 9), (62, 68), (62, 98), (23, 66), (82, 17), (19, 31), (30, 24), (70, 0), (30, 15), (96, 2), (11, 12), (5, 95), (66, 85), (95, 55), (50, 87), (75, 67), (91, 20), (17, 18), (18, 46), (53, 56), (9, 73), (74, 13), (13, 33), (1, 95), (82, 13), (32, 25), (3, 57), (3, 11), (59, 79), (42, 20), (4, 60), (3, 88), (12, 94), (16, 28), (12, 20), (5, 96), (13, 89), (4, 37), (38, 4), (2, 95), (13, 53), (19, 34), (21, 70), (40, 35), (13, 10), (68, 86), (78, 29), (80, 28), (3, 32), (13, 35), (78, 24), (43, 51), (74, 63), (2, 77), (1, 8), (31, 60), (77, 1), (99, 68), (62, 23), (63, 53), (50, 88), (77, 35), (20, 26), (4, 8), (14, 1), (29, 35), (70, 31), (76, 72), (31, 77), (3, 68), (35, 10), (34, 43), (15, 4), (23, 7), (77, 96), (34, 45), (19, 32), (35, 75), (35, 41), (83, 44), (91, 11), (65, 60), (78, 30), (3, 23), (1, 64), (31, 45), (50, 3), (89, 53), (33, 63), (2, 35), (20, 11), (62, 45), (1, 0), (10, 4), (42, 91), (3, 87), (22, 18), (29, 4), (24, 29), (1, 71), (52, 50), (55, 1), 
numberIntersection=3761 
minAngle=0.0 
sumAngle=34.06886355146432 
minEdgeLength=7.0 
sumEdgeLength=20725.142284463607 
minDistance=0.0 
sumDistance=1752242.8688166447 

-3761.0000000000 0.0000000000 34.0688635515 7.0000000000 20725.1422844636 0.0000000000 1752242.8688166447
0: (133, 200),
1: (133, 207),
2: (133, 215),
3: (133, 224),
4: (133, 237),
5: (133, 258),
6: (167, 200),
7: (167, 207),
8: (122, 267),
9: (167, 214),
10: (107, 267),
11: (167, 221),
12: (167, 229),
13: (93, 267),
14: (78, 267),
15: (67, 258),
16: (167, 238),
17: (167, 247),
18: (167, 259),
19: (67, 237),
20: (167, 272),
21: (67, 224),
22: (200, 200),
23: (167, 289),
24: (67, 215),
25: (167, 311),
26: (162, 333),
27: (146, 333),
28: (131, 333),
29: (67, 207),
30: (117, 333),
31: (67, 200),
32: (103, 333),
33: (90, 333),
34: (67, 193),
35: (67, 185),
36: (76, 333),
37: (67, 176),
38: (67, 163),
39: (61, 333),
40: (46, 333),
41: (33, 325),
42: (33, 299),
43: (33, 280),
44: (33, 265),
45: (67, 142),
46: (200, 284),
47: (135, 400),
48: (33, 253),
49: (33, 242),
50: (33, 233),
51: (33, 225),
52: (33, 218),
53: (33, 210),
54: (33, 203),
55: (33, 197),
56: (33, 190),
57: (33, 182),
58: (33, 175),
59: (33, 167),
60: (78, 133),
61: (33, 158),
62: (93, 133),
63: (33, 147),
64: (107, 133),
65: (122, 133),
66: (33, 135),
67: (33, 120),
68: (133, 142),
69: (33, 101),
70: (133, 163),
71: (33, 75),
72: (0, 373),
73: (46, 67),
74: (61, 67),
75: (76, 67),
76: (0, 236),
77: (133, 176),
78: (90, 67),
79: (103, 67),
80: (117, 67),
81: (131, 67),
82: (146, 67),
83: (162, 67),
84: (167, 89),
85: (0, 164),
86: (167, 111),
87: (167, 128),
88: (167, 141),
89: (167, 153),
90: (0, 27),
91: (167, 162),
92: (135, 0),
93: (167, 171),
94: (167, 179),
95: (167, 186),
96: (133, 185),
97: (200, 116),
98: (167, 193),
99: (133, 193),
133 200 133 207 133 215 133 224 133 237 133 258 167 200 167 207 122 267 167 214 107 267 167 221 167 229 93 267 78 267 67 258 167 238 167 247 167 259 67 237 167 272 67 224 200 200 167 289 67 215 167 311 162 333 146 333 131 333 67 207 117 333 67 200 103 333 90 333 67 193 67 185 76 333 67 176 67 163 61 333 46 333 33 325 33 299 33 280 33 265 67 142 200 284 135 400 33 253 33 242 33 233 33 225 33 218 33 210 33 203 33 197 33 190 33 182 33 175 33 167 78 133 33 158 93 133 33 147 107 133 122 133 33 135 33 120 133 142 33 101 133 163 33 75 0 373 46 67 61 67 76 67 0 236 133 176 90 67 103 67 117 67 131 67 146 67 162 67 167 89 0 164 167 111 167 128 167 141 167 153 0 27 167 162 135 0 167 171 167 179 167 186 133 185 200 116 167 193 133 193 0: (133, 200), 1: (133, 207), 2: (133, 215), 3: (133, 224), 4: (133, 237), 5: (133, 258), 6: (167, 200), 7: (167, 207), 8: (122, 267), 9: (167, 214), 10: (107, 267), 11: (167, 221), 12: (167, 229), 13: (93, 267), 14: (78, 267), 15: (67, 258), 16: (167, 238), 17: (167, 247), 18: (167, 259), 19: (67, 237), 20: (167, 272), 21: (67, 224), 22: (200, 200), 23: (167, 289), 24: (67, 215), 25: (167, 311), 26: (162, 333), 27: (146, 333), 28: (131, 333), 29: (67, 207), 30: (117, 333), 31: (67, 200), 32: (103, 333), 33: (90, 333), 34: (67, 193), 35: (67, 185), 36: (76, 333), 37: (67, 176), 38: (67, 163), 39: (61, 333), 40: (46, 333), 41: (33, 325), 42: (33, 299), 43: (33, 280), 44: (33, 265), 45: (67, 142), 46: (200, 284), 47: (135, 400), 48: (33, 253), 49: (33, 242), 50: (33, 233), 51: (33, 225), 52: (33, 218), 53: (33, 210), 54: (33, 203), 55: (33, 197), 56: (33, 190), 57: (33, 182), 58: (33, 175), 59: (33, 167), 60: (78, 133), 61: (33, 158), 62: (93, 133), 63: (33, 147), 64: (107, 133), 65: (122, 133), 66: (33, 135), 67: (33, 120), 68: (133, 142), 69: (33, 101), 70: (133, 163), 71: (33, 75), 72: (0, 373), 73: (46, 67), 74: (61, 67), 75: (76, 67), 76: (0, 236), 77: (133, 176), 78: (90, 67), 79: (103, 67), 80: (117, 67), 81: (131, 67), 82: (146, 67), 83: (162, 67), 84: (167, 89), 85: (0, 164), 86: (167, 111), 87: (167, 128), 88: (167, 141), 89: (167, 153), 90: (0, 27), 91: (167, 162), 92: (135, 0), 93: (167, 171), 94: (167, 179), 95: (167, 186), 96: (133, 185), 97: (200, 116), 98: (167, 193), 99: (133, 193), 
Time taken: 0.067 s
